File: totosuki/ABC/B/074.py
N = int(input())
K = int(input())
x = list(map(int, input().split()))

rslt = 0
for i in range(N):
  if abs(K - x[i]) > x[i]:
    rslt += x[i] * 2
  else:
    rslt += abs(K - x[i]) * 2

print(rslt)

File: OCH_Tao/ABC/B/074.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input[0], 10);
const K = parseInt(input[1], 10);
const X = (input[2].split(" ")).map(x => parseInt(x, 10));
let l = [];
for (let i = 0; i < N; i++) {
  l.push(2 * (Math.min(X[i], K - X[i])));
}
console.log(l.reduce((a, b) => a + b));